app/controllers/v0/messages_controller.rb

```diff
@@ -61,6 +51,22 @@ def thread
              meta: resource.metadata
     end
 
+    def reply
+      message = Message.new(message_params)
+
+      if message.body.blank?
+        message.errors.add(:body, "can't be blank")
+        raise Common::Exceptions::ValidationErrors, message
+      end
+
+      resource = client.post_create_message_reply(params[:id], message_params)
+      raise VA::API::Common::Exceptions::RecordNotFound, params[:id] unless resource.present?
+
+      render json: resource,
+             serializer: MessageSerializer,
+             status: :created
+    end
+
     def categories
       resource = client.get_categories
       raise VA::API::Common::Exceptions::InternalServerError unless response.present?
```

Comment:
Lets get rid of this, I missed this before -- the ClientResponseError in lib is handled, raising an InternalServerError is bad practice.